storage/src/main/java/org/apache/uniffle/storage/handler/impl/ComposedClientReadHandler.java

```diff
@@ -46,7 +46,22 @@ public class ComposedClientReadHandler extends AbstractClientReadHandler {
   private static final Logger LOG = LoggerFactory.getLogger(ComposedClientReadHandler.class);
 
   private enum Tier {
-    HOT, WARM, COLD, FROZEN;
+    /**
+     * data in memory
+     */
+    HOT,
+    /**
+     * data in local disk
+     */
+    WARM,
+    /**
+     * data in remote storage
+     */
+    COLD,
+    /**
+     * not supported temporarily
+     */
+    FROZEN;
```

Comment:
I read the code, and the comments here are not sure whether it is reasonable. PTAL @jerqi

Reply:
This is not 100% true, because some storage tier may be omitted, for example `HOT=MEM, WARM=HDFS`.

Reply:
Sorry for forgetting it, do you have some good ideas here?